DEVELOPMENT.md

```diff
@@ -0,0 +1,178 @@
+# How to work on Buzz About
+
+This is a fairly comprehensive guide to setting up a development environment, aimed at new students. You may already have some of the necessary programs installed&mdash;if so, you can just skip those steps!
+
+This document is draws from prior writeups by [Paul Friederichsen][] and [Prof. Nic McPhee][].
+
+**Table of Contents**
+
+1. Creating a GitHub Account
+2. Installing Visual Studio Code
+3. Installing `git`
+   1. On Windows
+   2. On a Mac
+4. Installing GitKraken
+5. Installing `npm`
+6. Telling `npm` how to talk to GitHub Packages
+   1. Generating a personal access token
+   2. Giving that personal access token to `npm`
+<!-- 7. Installing all of the third-party libraries -->
```

Comment:
You should be able to generate this automatically with a VS Code extension [like this one](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one). Regardless, the TOC should have section links.

Reply:
I had no idea that was possible! Thank you!